### ISSUE TITLE
Caused by: java.lang.ClassNotFoundException: org.apache.camel.impl.debugger.BacklogDebugger

### ISSUE THREAD

**Issue by biko8** [ncasaux/camel-plantuml#16]
With camel version of 4.2.0, Im having this error

```
Caused by: java.lang.ClassNotFoundException: org.apache.camel.impl.debugger.BacklogDebugger
```

This version of camel have this implementation inside `org.apache.camel.spi.BacklogDebugger`

**Comment by ncasaux** [comment]
Thanks for your interest in this project!

I just published a 1.7.0 version which should be soon available in maven central. So don't hesitate to let me know it works as expected! 😃

**Comment by biko8** [comment]
Thanks @ncasaux but in our project its impossible to update camel, we will try to do it as soon as possible.
